draft-irtf-cfrg-opaque.md

```diff
@@ -1020,36 +1027,35 @@ Specifically, a one-time-pad is derived from Ke2 and then used as input to XOR
 with the plaintext. In pseudocode, this is done as follows:
 
 ~~~
-info_pad = HKDF-Expand(Ke2, "encryption pad", len(server_info))
+info_pad = Expand(Ke2, "encryption pad", len(server_info))
 enc_server_info = xor(info_pad, server_info)
 ~~~
 
 # Configurations {#configurations}
 
-An OPAQUE configuration is a tuple (OPRF, Hash, MHF, EnvelopeMode, Group). The OPAQUE
-OPRF protocol is drawn from the "base mode" variant of {{I-D.irtf-cfrg-voprf}}. The
-following OPRF ciphersuites are supported:
-
-- OPRF(ristretto255, SHA-512)
-- OPRF(decaf448, SHA-512)
-- OPRF(P-256, SHA-256)
-- OPRF(P-384, SHA-512)
-- OPRF(P-521, SHA-512)
+An OPAQUE-3DH configuration is a tuple (OPRF, KDF, MAC, MHF, Group).
+The OPRF protocol is drawn from the "base mode" variant of {{I-D.irtf-cfrg-voprf}}.
+The KDF functions in this document use HKDF {{!RFC5869}}. The MAC functions in
+this document use HMAC {{!RFC2104}}. The Hash functions in this specification
+include SHA-256 and SHA-512. The MHF functions use Scrypt {{?RFC7914}}
```

Comment:
why not argon2i?

Reply:
One can certainly use argon2i, but this document (right now) only uses scrypt.

Reply:
that is clear. my question was why?

Reply:
This is not a great answer, but.... we can't properly cite Argon2i as it's [not yet an RFC](https://datatracker.ietf.org/doc/draft-irtf-cfrg-argon2/). (The same is true of the OPRF draft, but that's a dependency we can't workaround.) 

We can certainly change this to argon2i, or whatever, provided there's a suitable reference and evidence that the new thing is more widely supported than scrypt. The purpose of this is to be a recommendation for applications should they not know what they're doing.